tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -936,17 +932,17 @@ def map(self, map_func, num_parallel_calls=None):
     For example:
 
     ```python
-    # NOTE: The following examples use `{ ... }` to represent the
-    # contents of a dataset.
-    a = { 1, 2, 3, 4, 5 }
+    a = Dataset.range(1, 6)  # [ 1, 2, 3, 4, 5 ]
 
-    a.map(lambda x: x + 1) = { 2, 3, 4, 5, 6 }
+    a.map(lambda x: x + 1)  # ==> [ 2, 3, 4, 5, 6 ]
     ```
 
     The input signature of `map_func` is determined by the structure of each
     element in this dataset. For example:
 
     ```python
+    # NOTE: The following examples use `{ ... }` to represent the
```

Comment:
why do you introduce this comment here? I thought the purpose of this PR was to replace `{ ... }` with code that runs

Reply:
There are several examples following line in line 944 - 961 that I didn't change in this request and still contain non executable code. I wanted to keep this request small because I expected some feedback. I'll take another look in a separate request.